solaris/solaris10/generate_wazuh_packages.sh

```diff
@@ -208,6 +209,10 @@ clean(){
     groupdel ossec
 }
 
+ctrl_c() {
+    clean
```

Comment:
Killing the process with a SIGKILL must return anything distinct from 0.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/8258f302aceafdf80b87bebecb7a09803cbcdfb0.